common/psbt_open.c

```diff
@@ -471,3 +471,17 @@ bool psbt_has_required_fields(struct wally_psbt *psbt)
 
 	return true;
 }
+
+#if EXPERIMENTAL_FEATURES
+void psbt_input_set_final_witness_stack(struct wally_psbt_input *in,
+					struct witness_element **elements)
```

Comment:
const struct witness_element **?

Reply:
sort of a rough thing because the struct these are embedded in is generated code and doesn't include the const (but probably should?).